api_reference/requirements.txt

```diff
@@ -1,6 +1,7 @@
 myst_parser
 sphinx==7.3.7
 pydata-sphinx-theme
-arize-phoenix-evals
+linkify-it-py
+packages/phoenix-evals
```

Comment:
resolves #3722 ?

Reply:
As in you have to ship it before you know? This might work? https://github.com/langchain-ai/langchain/blob/aa165539f6cc43b0dedb036ec2dd751f5373e3c0/docs/api_reference/requirements.txt#L4

Reply:
This works but wondering if this addresses the concern for building from source